src/net/relay.rs

```diff
@@ -1795,6 +1932,22 @@ impl Relayer {
                             }
                         }
                     }
+
+                    // have the p2p thread forward all new unconfirmed microblocks obtained through tip sync protocol
+                    if let Some((id, mbs)) = new_synced_microblocks {
+                        if mbs.len() > 0 {
+                            debug!("{:?}: Microblocks from tip sync protocol: {}", &_local_peer, mbs.len());
+                            let mblocks_msg = MicroblocksData {
+                                index_anchor_block: id,
+                                microblocks: mbs,
+                            };
+
+                            let msg = StacksMessageType::Microblocks(mblocks_msg);
+                            if let Err(e) = self.p2p.broadcast_message(vec![], msg) {
```

Comment:
I'm not sure we should be doing this here.  These microblocks have _already_ been broadcasted -- there's a good chance that most or all of the neighbors already have them, so re-sending them is likely to be wasteful.  The microblock anti-entropy protocol is meant for nodes that for whatever reason didn't receive the microblocks to recover from a local lack of data.

Reply:
Removed.